README.md

```diff
@@ -1,2 +1,116 @@
 # box
-a preconfigured Vagrant box with a full GCC ARM toolchain for embedded development
+
+``uwmidsun/box`` is a preconfigured Vagrant box with a full GCC ARM toolchain for compiling ARM code and flashing and debugging it with tools like OpenOCD and gdb. Using this virtual environment, you can get set up to compile and load ARM code from any platform that Vagrant supports (Windows, Mac OSX, Linux).
+
+No provisioning tools or setup is really even required to use ``uwmidsun/box``. Since everything is packaged into the base box, running ``vagrant`` is super fast, you'll never have to worry about your environment breaking with updates, and you won't need the internet to code.
+
+The ARM toolchain that will be set up includes the following tools:
+
+* GCC ARM Embedded
```

Comment:
The Commonmark md style guides I've seen call for three spaces like:
```*···Words```

I honestly don't care that much just a suggestion.